cautils/reportv2tov1.go

```diff
@@ -120,7 +120,11 @@ func controlReportV2ToV1(opaSessionObj *OPASessionObj, frameworkName string, con
 						ruleResponse := reporthandling.RuleResponse{}
 						ruleResponse.Rulename = rulev2.GetName()
 						for i := range rulev2.Paths {
-							ruleResponse.FailedPaths = append(ruleResponse.FailedPaths, rulev2.Paths[i].FailedPath)
+							if rulev2.Paths[i].FailedPath != "" {
+								ruleResponse.FailedPaths = append(ruleResponse.FailedPaths, rulev2.Paths[i].FailedPath)
+							} else if rulev2.Paths[i].FixPath.Path != "" {
```

Comment:
why else? what will happen both paths are not empty..

Reply:
not a case thats supposed to happen- it should always be either failPath or FixPath. but i'll change it just in case